test/cypress/integration/entry-editor-extension.spec.js

```diff
@@ -0,0 +1,34 @@
+import { entry } from '../utils/paths'
+
+const post = {
+  id: '5mwUiJB2kThfAG9ZnRNuNQ',
+  title: 'My post with a custom entry editor',
+  body: ''
```

Comment:
Might be worth actually adding a `body` in as it may catch false-positives where something else renders with an empty string.